AppiumForMac/Server/Handlers/AfMHandlers.m

```diff
@@ -601,7 +601,7 @@ - (AppiumMacHTTPJSONResponse *)post_element_elements:(NSString*)path data:(NSDat
         AfMElementLocator *locator = [AfMElementLocator locatorWithSession:session using:using value:value];
         
         // initialize status as though no element were found
-        *statusCode = kAfMStatusCodeNoSuchElement;
+        *statusCode = kAfMStatusCodeSuccess;
         
         if (locator != nil)
```

Comment:
can we avoid the code duplication here?

Reply:
If you mean duplicating with the code in findAllUsingBaseUIElement I think I can. But the issue is that it's not 100 % duplicated. Since the findAllUsingBaseUIElement did not handle all scenarios where in the code block where it handles other locators besides xpath it does not assign a statusCode. So i'd have to move this line over to after
https://github.com/appium/appium-for-mac/blob/f7036da9cf37c463ae4cb139fa8a4e9391619fb3/AppiumForMac/Server/Controller/AfMElementLocator.m#L261
This line for it to work.